testsuite-real/test-btrfsutils.cc

```diff
@@ -0,0 +1,106 @@
+
+
+#include <errno.h>
+#include <fcntl.h>
+#include <string.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#include <iostream>
+
+#include "snapper/BtrfsUtils.h"
+
+
+using namespace std;
+using namespace snapper;
+using namespace BtrfsUtils;
+
+
+int
+main()
+{
+    int fd = open("/btrfs", O_NOATIME);
+    if (fd < 0)
+    {
+	cerr << "open failed (" << strerror(errno) << ")" << endl;
+	return EXIT_FAILURE;
+    }
+
+
+    if (false)
```

Comment:
Since these are integration tests and the program must be adapted. I added a comment.